content/applications/sales/point_of_sale/overview/customer_notes.rst

```diff
@@ -0,0 +1,38 @@
+==============
+Customer notes
+==============
+
+**Customer notes** allow you to provide customers with further information on a specific product.
+For instance, you can give them cleaning and maintenance tips, etc. They can also be used to track a
+customer’s particular requests. For example, they do not want the product to be assembled, etc.
+
+Add customer notes
+==================
+
+You can add notes directly from an open :ref:`POS session <pos/start-session>`. To do so, click
+:guilabel:`Customer Notes` on the pad. Doing so opens a pop-up window on which you can add the
+desired content for the note.
+
+.. note::
+   Product notes from an :doc:`imported SO <../shop/sales_order>` are displayed identically in the
+   cart.
+
+.. image:: customer_notes/customer-notes-ui.png
+   :align: center
+   :alt: customer note button and notes (SO and POS session) on products in the cart
+
+Modify a customer note
+======================
+
+Select the product from the cart to modify an existing note and click :guilabel:`Customer Notes`.
```

Comment:
```suggestion
Select the product from the cart to modify an existing note and click :guilabel:`Customer Notes`.
```
To modify an existing note, select the product and click :guilabel:`Customer Note`.